OpenRA.Mods.Common/Widgets/Logic/Editor/NewMapLogic.cs

```diff
@@ -68,26 +68,30 @@ public NewMapLogic(Action onExit, Action<string> onSelect, Ruleset modRules, Wid
 				map.PlayerDefinitions = new MapPlayers(map.Rules, map.SpawnPoints.Value.Length).ToMiniYaml();
 				map.FixOpenAreas(modRules);
 
-				var userMapFolder = Game.ModData.Manifest.MapFolders.First(f => f.Value == "User").Key;
+				Action<string> afterSave = uid =>
+				{
+					// HACK: Work around a synced-code change check.
```

Comment:
Why is this a hack and why can't you reproduce it? It happens almost all the time for me.